sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md

```diff
@@ -149,6 +149,9 @@
 ### Major changes
 - SecretClient.vaultUrl has been added with the original value pass to SecretClient.
 
+## 4.0.0-preview.3 (2019-09-10)
```

Comment:
I don't know why the date is later than the previous version but we should have released `4.0.0-preview.3` before `4.0.0-preview.4`.